asapdiscovery-data/asapdiscovery/data/schema_v2/target.py

```diff
@@ -171,3 +172,17 @@ def from_oedu_file(cls, oedu_file: Union[str, Path], **kwargs) -> "PreppedTarget
     def to_oedu_file(self, filename: Union[str, Path]) -> None:
         oedu = self.to_oedu()
         save_openeye_design_unit(oedu, filename)
+
+    def to_pdb_file(self, filename: str):
+        """
+        Write the prepared target receptor to PDB file using openeye.
+
+        Parameters
+        ----------
+        filename: The name of the pdb file the target should be writen to.
+
+        """
+        oe_receptor = oechem.OEGraphMol()
+        oedu = self.to_oedu()
+        oedu.GetProtein(oe_receptor)
+        save_openeye_pdb(oe_receptor, pdb_fn=filename)
```

Comment:
in the past it's been necessary to "re-perceive" residues in the protein (after adding hydrogens) otherwise CONECT records and atom ordering gets wonky. This is the function we've used to do that:
https://github.com/choderalab/asapdiscovery/blob/c48443f9632a84f57df7f9fa7ede16174cdc49c3/asapdiscovery-data/asapdiscovery/data/openeye.py#L526

I don't think it would hurt to add and it could definitely help, esp with CONECT records for later simulations.

in the past we've used this function to split an oedu into the protein / ligand components (although not much different functionality than what you've done here, just wanted to make sure you saw it).
https://github.com/choderalab/asapdiscovery/blob/c48443f9632a84f57df7f9fa7ede16174cdc49c3/asapdiscovery-modeling/asapdiscovery/modeling/modeling.py#L732

Reply:
Ah thanks, thats good to know about the perceive residues, I wonder if would there be any harm in doing this by default in the `save_openeye_pdb` function? No I missed the split function I'll switch to using that!